lambda.js

```diff
@@ -11,7 +11,7 @@ const jobs = require('./build/server/workers/job-processes')
 // See: http://docs.aws.amazon.com/lambda/latest/dg/best-practices.html#function-code
 // "Separate the Lambda handler (entry point) from your core logic"
 
-exports.handler = (event, context) => {
```

Comment:
lets put a PSA comment here that lambda doesnt know its actually done (and may retry on perceived failure) unless handleCallback is called (eek)